src/index.php

```diff
@@ -0,0 +1,43 @@
+<?php require_once 'php.php';
+$collection= fetchPigmentData()
+
+
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="modern-normalize.css">
+    <link rel="stylesheet" href="stylesheet.css">
+    <meta name = "viewpoint" content = "width = device - width, initial-scale=1.0">
```

Comment:
I was previously corrected that it should be "viewport" not "viewpoint"

Reply:
thanks for the tip!